src/TendoPay/Exceptions/TendoPay_Integration_Exception.php

```diff
@@ -23,5 +23,54 @@ class TendoPay_Integration_Exception extends \Exception {
 	 */
 	public function __construct( $message, \Throwable $previous = null ) {
 		parent::__construct( $message, 0, $previous );
+
+		try {
+			$this->report($message, $previous);
+		} catch(Exception $e) {}
```

Comment:
Please log this exception to the regular (local) log file, since it can provide more details on what went wrong with the log broadcasting feature (if we'll have access to the log files)

Reply:
I'll leave a filelog like uploads/wc-logs/fatal-errors-2018-11-19-5bf3571e7d9510.25430770